frontend/packages/ux-editor/src/components/TextResource.tsx

```diff
@@ -77,6 +81,11 @@ export const TextResource = ({
     }
   };
 
+  const handleDeleteButtonClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    event.stopPropagation();
```

Comment:
Why do we need to stopPropagation here? :) If it's required, do you think we should add a comment on why?

Reply:
No reason, just some leftover code 😅